frontend/src/components/SecretRow.vue

```diff
@@ -5,112 +5,113 @@ SPDX-License-Identifier: Apache-2.0
 -->
 
 <template>
-  <v-list-item>
-    <v-list-item-content>
-      <v-list-item-title class="mb-1">
-        {{secret.metadata.name}}
-        <v-tooltip v-if="!isOwnSecret" top>
+  <tr>
+    <td v-if="selectedHeaders.name">
+      <div class="d-flex">
+        {{item.name}}
+        <v-tooltip v-if="!item.isOwnSecret" top>
           <template v-slot:activator="{ on }">
             <v-icon v-on="on" small class="mx-1">mdi-account-arrow-left</v-icon>
           </template>
-          <span>Secret shared by {{secretOwner}}</span>
+          <span>Secret shared by {{item.ownernamespace}}</span>
```

Comment:
`ownernamespace` should be camelCase `ownerNamespace`

Reply:
done